plugin/front/ticket.form.php

```diff
@@ -39,15 +46,26 @@
 } elseif (!empty($update)) {
     $ticket->check($id, UPDATE, $post);
 
-    $post = PluginIserviceTicket::preProcessPostData($post);
-
     $ticket->update($post);
 
     $ticket->updateItem($id, $post);
 
     Html::redirect($ticket->getFormURL() . '?mode=9999&id=' . $id);
 }
 
+if (!empty($addConsumable) && !empty($id)) {
+    $ticket->addConsumable($id, $post);
+}
+
+if (!empty($removeConsumable) && !empty($id)) {
+    $ticket->removeConsumable($id, $post);
+}
+
+if (!empty($updateConsumable) && !empty($id)) {
+    $ticket->updateConsumable($id, $post);
+}
```

Comment:
If these cannot be true at once (which they can't), use `if () {} elseif () {}`
```suggestion
if (!empty($addConsumable) && !empty($id)) {
    $ticket->addConsumable($id, $post);
} elseif (!empty($removeConsumable) && !empty($id)) {
    $ticket->removeConsumable($id, $post);
} elseif (!empty($updateConsumable) && !empty($id)) {
    $ticket->updateConsumable($id, $post);
}
```
The order should be: the most probable first.